assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +355,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
```

Comment:
Yes, the host that the allocator generates is in that format, however it's meant to be pluggable so I'm not sure it's something we want to rely on in a check elsewhere.  /cc @ramr 

Also correct that it is possible to have the same host for a secure and non-secure route as a valid use case.  There is a filter called `routeWebURL` that can be used to determine scheme based on the tls termination type.  Once we know the protocol of the route (future) we should be able to make that smarter as well.

The path is also important since someone may be making routes like www.example.com/secure that is secured while www.example.com/public is not.